### DISCUSSION TITLE
پایتون فرگمنت

### DISCUSSION THREAD

**Discussion by chakkal9877** [GFW-knocker/gfw_resist_tls_proxy#84] [General]
برای وصل شدن به اینترنت، از همین پایتون خیلی ضعیف عمل میکنه نسبت به قبل، با مخابرات پینگ 1500 ms میداد ، الان پینگ ضعیف میده و سرعت دانلود به شدت پایین اومده
چه حرکتی باید زد؟

**Comment by GFW-knocker** [comment]
فرگمنت فیلتر sni رو دور میزنه
به نظر میرسه پهنای باند کل کشور رو کاهش دادن امروز
شما ایپی های دیگه کلودفلر رو تست کنید
و فرگمنت randchank تاخیر کمتری داره

**Comment by chakkal9877** [comment]
قبلا تست میکردم خیلی خوب بود و ثابت روی 1500ms الان ایپی های دیگرو هم میزنم
باز هم تغییراتی روی L fragment
میزنم ، عددشو بالا پایین میدم، گاهی به زیر 1000ms  میاد ولی ثابت و پایدار نیست

برای این اتصال هم میشه فکری کنید؟
تو یوتیوب بخوبی متصل شد و واقعا عملکرد خوبی دیدم

البته اپلود رو تست نکردم
